synapse/handlers/oidc_handler.py

```diff
@@ -497,7 +497,10 @@ async def _parse_id_token(self, token: Token, nonce: str) -> UserInfo:
         return UserInfo(claims)
 
     async def handle_redirect_request(
-        self, request: SynapseRequest, client_redirect_url: bytes
+        self,
+        request: SynapseRequest,
+        client_redirect_url: bytes,
+        ui_auth_session_id: Optional[str] = None,
     ) -> None:
```

Comment:
this now returns a `str` rather than a None? could do with updating the docstring too.

Reply:
Good catch. Unfortunate mypy didn't see this. 😮